torch/csrc/jit/serialization/source_range_serialization.cpp

```diff
@@ -65,10 +91,36 @@ c10::IValue SourceRangeSerializer::serialize_source(
   }
   c10::intrusive_ptr<c10::ivalue::Tuple> serialized;
   if (s == nullptr) {
-    serialized = c10::ivalue::Tuple::create({"", "", 0});
+    serialized = c10::ivalue::Tuple::create({-1, -1, 0});
   } else {
+    auto text_iter = text_to_pos_.find(s->text());
```

Comment:
Nit:
rename `text_to_pos_` to `text_to_idx_` because `pos` has another meaning in the context of source range.

Reply:
Another nit:
This chunk of logic might be a good fit for a utility func, so that it is more readable and reusable by the fname caching logic below

Reply:
done.